openml/datasets/functions.py

```diff
@@ -815,12 +815,16 @@ def edit_dataset(
 ) -> int:
     """
       Edits an OpenMLDataset.
-      Specify atleast one field to edit, apart from data_id
-       - For certain fields, a new dataset version is created : attributes, data,
-       default_target_attribute, ignore_attribute, row_id_attribute.
+      Specify atleast one field to edit, apart from data_id.
 
-       - For other fields, the uploader can edit the exisiting version.
-        Noone except the uploader can edit the exisitng version.
+      This API has non-critical and critical fields.
```

Comment:
I'd word it as `This function allows editing of both non-critical and critical fields.`.